modules/anu_lms_assessments/src/Form/AssessmentQuestionResultSettingsForm.php

```diff
@@ -46,7 +46,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
    *   Form definition array.
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $form['assessmentquestionresult_settings']['#markup'] = 'Settings form for Quiz question result entities. Manage field settings here.';
+    $form['assessmentquestionresult_settings']['#markup'] = t('Settings form for Quiz question result entities. Manage field settings here.');
```

Comment:
@ikorgik is this text that needs to be added to the LMS translation spreadsheets?

Reply:
No, it's for admins